temp_streaming_function.py

```
@@ -0,0 +1,394 @@
+                # Check if simulation is already started - if so, ignore the begin command
+                if orchestrator.state.simulation_started:
+                    print(f"[STREAM DEBUG] Simulation already started, ignoring 'begin' command")
+                    # Return a message saying simulation is already running
+                    already_started_msg = "The simulation is already in progress. You can continue interacting with the personas."
+                    for char in already_started_msg:
+                        yield f"data: {json.dumps({'content': char, 'done': False})}\n\n"
+                        await asyncio.sleep(0.02)
+                    yield f"data: {json.dumps({'done': True, 'persona_name': 'ChatOrchestrator', 'persona_id': None, 'scene_completed': False, 'next_scene_id': None, 'turn_count': orchestrator.state.turn_count, 'full_content': already_started_msg})}\n\n"
+                    return
+                
+                last_msg = db_session.query(ConversationLog).filter(
```

Ruff (0.14.1)

Undefined name `ConversationLog`

(F821)